docs/quick-start/publish-site-to-aws.mdx

```diff
@@ -10,8 +10,7 @@ Publish a site to AWS using S3 snapshots.
 ## Step 1: Generate a Mosaic site
 
 ```
-> yarn global add @jpmorganchase/mosaic-create-site
-> mosaic-create-site create -o my-sample-site
+> npx @jpmorganchase/mosaic-create-site -o my-sample-site
```

Comment:
I tried to do this in a GitHub codespace and got this error:

$ npx @jpmorganchase/mosaic-create-site create -o my-sample-site
node:internal/modules/cjs/loader:1090
  const err = new Error(message);
              ^

Error: Cannot find module '/usr/local/share/nvm/versions/node/v19.9.0/lib/node_modules/@jpmorganchase/mosaic-standard-generator/dist/generator.js'
Require stack:
- /home/codespace/.npm/_npx/42ed83063eb32b24/node_modules/resolve-global/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1090:15)
    at Function.resolve (node:internal/modules/helpers:125:19)
    at resolveGlobal (/home/codespace/.npm/_npx/42ed83063eb32b24/node_modules/resolve-global/index.js:9:18)
    at resolvePath (file:///home/codespace/.npm/_npx/42ed83063eb32b24/node_modules/@jpmorganchase/mosaic-create-site/dist/create.mjs:37:20)
    at file:///home/codespace/.npm/_npx/42ed83063eb32b24/node_modules/@jpmorganchase/mosaic-create-site/dist/create.mjs:47:27
    at Array.map (<anonymous>)
    at createMosaicApp (file:///home/codespace/.npm/_npx/42ed83063eb32b24/node_modules/@jpmorganchase/mosaic-create-site/dist/create.mjs:45:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/codespace/.npm/_npx/42ed83063eb32b24/node_modules/resolve-global/index.js'
  ]
}

whereas doing it the old way worked (yarn global add @jpmorganchase/mosaic-create-site
then mosaic-create-site create -o my-sample-site)

maybe the npx way isn't as reliable or has more prequisites, could we suggest both? @mark-tate @DavieReid